client/my-sites/site-settings/form-general.jsx

```diff
@@ -413,10 +411,7 @@ export class SiteSettingsFormGeneral extends Component {
 							<FormRadio
 								name="blog_public"
 								value="-1"
-								checked={
-									( -1 === blogPublic && ! wpcomComingSoon && ! isPrivateAndUnlaunched ) ||
-									( wpcomComingSoon && isAtomicAndEditingToolkitDeactivated )
-								}
+								checked={ -1 === blogPublic }
```

Comment:
@ramonjd would really appreciate your feedback on this. I tested every combination I could come up with and found this predicate enough to determine whether "Private" should be ticked.a

Reply:
Thanks for cleaning this up @alshakero 🙇 , and sorry for the mashed potato conditions I left you. 

Those were the heady times of trying to juggle two competing coming soon versions. 😄 

What you have should cover the most important situations, if not all of them, and it looks like folks have confirmed that overnight anyway.

It made me dive into my memory to work out our rationale, which was mainly to display existing coming soon v1 sites (blogPublic == -1) as "coming soon" in the form UI, and it seemed to work at the time in our testing but...🤷 . 

The reason was that v1 sites, even though they were private, showed the coming soon page, so we wanted to make the state, at least in the setting form UI, consistent 😄 

Just jotting it down here because I don't have a pen and paper :)

```js

// check private if blogPublic is set to private AND the site isn't coming soon v1 or coming soon v1 unlaunched (in which case we show in the UI that it's coming soon)
( -1 === blogPublic && ! wpcomComingSoon && ! isPrivateAndUnlaunched ) ||
            // check private if it's a coming soon v1 atomic site AND the user has removed the editing toolkit (in which case coming soon wouldn't work at all, so we switch to private to "try" to honour the last state 
									( wpcomComingSoon && isAtomicAndEditingToolkitDeactivated )
```

Thanks again for working this out and making it better!